### PR TITLE
fix: don't console.error MISSING_TRANSLATION errors

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/suite/src/support/suite/ConnectedIntlProvider.tsx [hunk 1]
```diff
@@ -8,7 +8,17 @@ const ConnectedIntlProvider: React.FC = ({ children }) => {
         messages: state.suite.messages,
     }));
     return (
-        <IntlProvider locale={locale} messages={messages}>
+        <IntlProvider
+            locale={locale}
+            messages={messages}
+            onError={err => {
```
Comment (matejkriz) [inline packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: default error handler using `process.env.NODE_ENV !== 'production'` condition, see https://github.com/formatjs/formatjs/blob/930ed77c458bf740f6863839ad9d2f3c820b71c3/packages/intl/src/utils.ts#L29

So if you override it, please add that condition too...

Comment (mroz22) [reply packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: ok. updated

Comment (matejkriz) [reply packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: We use `import { isDev } from '@suite-utils/build';` and not process.env.NODE_ENV directly. We could discuss if it's useful, but let's keep it consistent.

Comment (matejkriz) [reply packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: sorry for nitpicking

Comment (mroz22) [reply packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: I know we do :D but from your previous reaction I thought you want it to be `process.env.NODE_ENV !== 'production'` literally :D

Comment (mroz22) [reply packages/suite/src/support/suite/ConnectedIntlProvider.tsx:15]: updated

**Review by matejkriz — APPROVED** [review]

**Review by tomasklim — APPROVED** [review]

**Pull request merged**